planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/include/autoware/behavior_path_static_obstacle_avoidance_module/scene.hpp

```diff
@@ -117,9 +118,44 @@ class StaticObstacleAvoidanceModule : public SceneModuleInterface
    */
   void updateRegisteredRTCStatus(const PathWithLaneId & path)
   {
+    // NOTE(odashima): Prevent duplication of control points of PlanningFactor.
+    auto filterCloseShiftLines =
+      [](const RegisteredShiftLineArray & shift_lines) -> RegisteredShiftLineArray {
+      constexpr double distance_threshold = 0.05;
+      RegisteredShiftLineArray filtered_lines;
+
+      for (size_t i = 0; i < shift_lines.size(); ++i) {
+        const auto & current_line = shift_lines[i];
```

Comment:
Please use `at()` function.
```suggestion
        const auto & current_line = shift_lines.at(i);
```